### PR TITLE
Prevent sticker hide on expanded view.

### PR DESCRIPTION
**Pull request by sarankumar-ns**
Fixed issue : 1 ) After previewed sticker in expanded mode selected sticker gone invisible.

2 ) Changed URL of WhatsApp Play Store Page.

### CODE REVIEW

**Comment by simonzhexu** [comment]
Could you provide a test plan of before and after, preferably with video?

Thanks for contributing.

**Comment by sarankumar-ns** [comment]
Before - https://youtube.com/shorts/sh3dJfs5SqM

After - https://youtube.com/shorts/0rZTylxK0R4

WhatsApp - https://youtube.com/shorts/X4kxcnBddOM

I think WhatsApp also not making that view invisible when expanded.

File: Android/app/src/main/java/com/example/samplestickerapp/AddStickerPackActivity.java [hunk 1]
```diff
@@ -123,7 +123,7 @@ private void launchWhatsAppPlayStorePage() {
                 final boolean smbAppInstalled = WhitelistCheck.isPackageInstalled(WhitelistCheck.SMB_WHATSAPP_PACKAGE_NAME, packageManager);
                 final String playPackageLinkPrefix = "http://play.google.com/store/apps/details?id=";
                 if (whatsAppInstalled && smbAppInstalled) {
-                    launchPlayStoreWithUri("https://play.google.com/store/apps/developer?id=WhatsApp+Inc.");
+                    launchPlayStoreWithUri("https://play.google.com/store/apps/developer?id=WhatsApp+LLC");
```
Comment (MarvinJWendt) [inline Android/app/src/main/java/com/example/samplestickerapp/AddStickerPackActivity.java:126]: This is not related to the PR description, but definitely has to be fixed. Good catch!

**Review by MarvinJWendt — APPROVED** [review]
The clicked stickers should not disappear. The PR adjusts the behavior so that it matches the behavior in WhatsApp.
Along the way, an old, invalid URL is renewed. 👍

File: Android/app/src/main/java/com/example/samplestickerapp/AddStickerPackActivity.java (hunk 1 above)
Comment (sarankumar-ns) [reply Android/app/src/main/java/com/example/samplestickerapp/AddStickerPackActivity.java:126]: Ya... I just find that after made PR and don't want to make it as separate Pull Request. So, just commit with that request.

**Comment by simonzhexu** [comment]
Thanks for fixing the bug. I did not notice.

**Pull request merged**